common/core/revision.c.in

```diff
@@ -0,0 +1,10 @@
+#include "common/core/version.h"
+
+static const struct revision _revision = {
+    .primary = '${PRIMARY_REVISION}',
```

Comment:
Would be nice to either put a link to the confluence page that explains what the primary and secondary is, or just put a little blurb here.

Reply:
I think I might put a blurb in the readme instead - I don't like having that sort of content in a generated file